tools/esdlcomp/esdlcomp.h

```diff
@@ -1087,6 +1087,13 @@ class EspMethodInfo
             tags = p->next;
             delete p;
         }
+
+        if (name_)
+           free(name_);
+        if (request_)
+           free(request_);
+        if (response_)
+           free(response_);
```

Comment:
@kenrowland good point, but outside the scope of this particular issue. If it makes sense, please open that jira.